src/authgss_hash.c

```diff
@@ -92,17 +92,13 @@ svc_rpc_gss_cmpf(const struct opr_rbtree_node *lhs,
 	return (1);
 }
 
-void
+static void
 authgss_hash_init()
 {
 	int ix, code = 0;
 
 	mutex_lock(&authgss_hash_st.lock);
 
-	/* once */
-	if (authgss_hash_st.initialized)
-		goto unlock;
-
```

Comment:
We need those checks to avoid double initialization. Whether that happens in real life is different! We should actually put this in initialization, then we don't need to hold the lock.